db/migrate/20141215175503_rename_via_script.rb

```diff
@@ -0,0 +1,285 @@
+    remove_index 'mappings', name: 'index_links_on_ontology_id'
+    add_index 'mappings', ['ontology_id'], unique: false, name: 'index_mappings_on_ontology_id'
+    remove_index 'mappings', name: 'index_links_on_source_id'
+    add_index 'mappings', ['source_id'], unique: false, name: 'index_mappings_on_source_id'
+    remove_index 'mappings', name: 'index_links_on_target_id'
+    add_index 'mappings', ['target_id'], unique: false, name: 'index_mappings_on_target_id'
+    remove_index 'structured_proof_parts', name: 'index_structured_proof_parts_on_link_version_id'
+    add_index 'structured_proof_parts', ['mapping_version_id'], unique: false, name: 'index_structured_proof_parts_on_mapping_version_id'
+    execute "ALTER TABLE ONLY symbols DROP CONSTRAINT entities_ontology_id_fk;"
+    execute "ALTER TABLE ONLY symbols ADD CONSTRAINT symbols_ontology_id_fk FOREIGN KEY (ontology_id) REFERENCES ontologies(id) ON DELETE CASCADE;"
+    execute "ALTER TABLE ONLY oops_responses_symbols DROP CONSTRAINT entities_oops_responses_entity_id_fk;"
+    execute "ALTER TABLE ONLY oops_responses_symbols ADD CONSTRAINT oops_responses_symbols_symbol_id_fk FOREIGN KEY (symbol_id) REFERENCES symbols(id);"
+    execute "ALTER TABLE ONLY oops_responses_symbols DROP CONSTRAINT entities_oops_responses_oops_response_id_fk;"
+    execute "ALTER TABLE ONLY oops_responses_symbols ADD CONSTRAINT oops_responses_symbols_oops_response_id_fk FOREIGN KEY (oops_response_id) REFERENCES oops_responses(id) ON DELETE CASCADE;"
+    execute "ALTER TABLE ONLY sentences_symbols DROP CONSTRAINT entities_sentences_entity_id_fk;"
+    execute "ALTER TABLE ONLY sentences_symbols ADD CONSTRAINT sentences_symbols_symbol_id_fk FOREIGN KEY (symbol_id) REFERENCES symbols(id) ON DELETE CASCADE;"
+    execute "ALTER TABLE ONLY sentences_symbols DROP CONSTRAINT entities_sentences_ontology_id_fk;"
+    execute "ALTER TABLE ONLY sentences_symbols ADD CONSTRAINT sentences_symbols_ontology_id_fk FOREIGN KEY (ontology_id) REFERENCES ontologies(id) ON DELETE CASCADE;"
+    execute "ALTER TABLE ONLY sentences_symbols DROP CONSTRAINT entities_sentences_sentence_id_fk;"
+    execute "ALTER TABLE ONLY sentences_symbols ADD CONSTRAINT sentences_symbols_sentence_id_fk FOREIGN KEY (sentence_id) REFERENCES sentences(id) ON DELETE CASCADE;"
+    execute "ALTER TABLE ONLY symbol_groups DROP CONSTRAINT entity_groups_ontology_id_fk;"
+    execute "ALTER TABLE ONLY symbol_groups ADD CONSTRAINT symbol_groups_ontology_id_fk FOREIGN KEY (ontology_id) REFERENCES ontologies(id) ON DELETE CASCADE;"
+    execute "ALTER TABLE ONLY symbol_mappings DROP CONSTRAINT entity_mappings_source_id_fk;"
+    execute "ALTER TABLE ONLY symbol_mappings ADD CONSTRAINT symbol_mappings_source_id_fk FOREIGN KEY (source_id) REFERENCES symbols(id) ON DELETE CASCADE;"
+    execute "ALTER TABLE ONLY symbol_mappings DROP CONSTRAINT entity_mappings_target_id_fk;"
```

Comment:
Metrics/LineLength: Line is too long. [92/80]<br>Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.